static/install.html

```diff
@@ -158,7 +163,9 @@ <h3 id="obtaining-signify">
             <h2 id="enabling-oem-unlocking">
                 <a href="#enabling-oem-unlocking">Enabling OEM unlocking</a>
             </h2>
-            <p>OEM unlocking needs to be enabled from within the operating system.</p>
+            <p>OEM unlocking needs to be enabled from within the operating system.
```

Comment:
It's usually grayed out because of Factory Reset Protection in the stock OS which requires being able to connect to the internet to change this.

Reply:
I missed this change in my update. Please ignore the patch.